org.scummvm.ScummVM.json

```diff
@@ -24,6 +24,45 @@
         "*.la", "*.a"
     ],
     "modules": [
+        {
+            "name": "fluidsynth",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FluidSynth/fluidsynth/archive/v2.0.0.tar.gz",
+                    "sha256": "5ca094efbe13fdb880cfd0431354a7b85daf7239c344939493a2aaeca4e59ed5"
+                }
+            ],
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DLIB_INSTALL_DIR=lib"
+            ]
+        },
+        {
+            "name": "fluid_soundfont",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://http.debian.net/debian/pool/main/f/fluid-soundfont/fluid-soundfont_3.1.orig.tar.gz",
```

Comment:
Is the real upstream unreliable or something?

Reply:
I couldn't find the real upstream. Debian downloaded the file from MuseScore, but the download link is gone. MuseScore still has files (with newer file formats) that are based on it.